cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -771,9 +779,9 @@ public void startClusterMgmtServices() throws CloudbreakException {
     @Override
     public int deployConfigAndStartClusterServices() throws CloudbreakException {
         try {
-            LOGGER.info("Deploying configuration and starting services");
+            LOGGER.info("Deploying configuration and restarting services");
             deployConfig();
-            return startServices();
+            return restartServices();
```

Comment:
is this called during the provisioning? It can be cause any not observed issues during the provision?

Reply:
This is called when the only for DH clusters when they are started after a resize of DL.
The intent was to refresh the SDX remote data context and re-configure the services when the DH is started.